rsts/concepts/flyte_console.rst

```diff
@@ -0,0 +1,232 @@
+.. _ui:
+
+How to Use Flyte UI
+===================
+
+Flyte UI is a web-based user interface for Flyte. It helps interact with Flyte objects and builds DAGs out of your workflows.
+
+With Flyte UI, you can:
+
+* Launch Execution
+* Launch Task
+* View Versioned Executions
+* Trigger Versioned Executions
```

Comment:
```suggestion
* View Versioned Tasks and Workflows
* Trigger Versioned Tasks and Workflows
```

Reply:
or
```suggestion
* View Versioned Entities
* Trigger Versioned Entities
```

Reply:
I agree we do not have versioned executions you can filter

Reply:
You are right. It has to be "tasks and workflows".